lib/BuildSystem/BuildSystem.cpp

```diff
@@ -2444,40 +2444,38 @@ class StaleFileRemovalCommand : public Command {
     computeFilesToDelete();
 
     bsci.getDelegate().commandStarted(this);
-    auto success = true;
```

Comment:
Not needed anymore, because the command will not fail if files cannot be removed.